core/src/main/java/tc/oc/pgm/kits/KitParser.java

```diff
@@ -168,8 +168,17 @@ public WalkSpeedKit parseWalkSpeedKit(Element el) throws InvalidXMLException {
   }
 
   public ClearItemsKit parseClearItemsKit(Element el) throws InvalidXMLException {
-    if ("".equals(el.getChildText("clear"))) return new ClearItemsKit(true);
-    if ("".equals(el.getChildText("clear-items"))) return new ClearItemsKit(false);
+    Element applyClear = el.getChild("clear");
+    if (applyClear != null && applyClear.hasAttributes()) {
+      boolean items = XMLUtils.parseBoolean(applyClear.getAttribute("items"), false);
+      boolean armor = XMLUtils.parseBoolean(applyClear.getAttribute("armor"), false);
+      boolean effects = XMLUtils.parseBoolean(applyClear.getAttribute("effects"), false);
+      return new ClearItemsKit(items, armor, effects);
+    } else {
+      // legacy
+      if ("".equals(el.getChildText("clear"))) return new ClearItemsKit(true, true, false);
```

Comment:
It will be extremely counter-intuitive if `<clear/>` clears items, however `<clear effects="false" />` does not, and neither does `<clear id="asdf" />` (the id does absolutely nothing here, but it's still something someone may try to do).

Keep the default items armor and effects consistent, not "legacy", simply make items and armor default to true, and effects default to false.

Also i'm pretty sure right now this isn't properly supported, as it seems like `el` is the parent and you search for a single `clear` child:
```xml
<kit>
    <clear items="true" armor="false" effects="false"/>
    <clear items="false" armor="true" effects="false"/>
</kit>
```
It should probably be supported even if it's a bit stupid

Reply:
This would become inconsistent with how the rest of the kits are applied like the Shield, Double Jump and Fly Kit.